pkg/controller/machinedeployment/controller_test.go

```diff
@@ -0,0 +1,91 @@
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+package machinedeployment_test
+
+import (
+	"time"
+
+	. "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+	. "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
Step 1 after this will be kill these two imports with a fire. In fact, could you add a commit that converts these tests to normal tests like the machine controller? @kcoronado did that and should be able to answer questions.

Reply:
removed the test for now.
i have a version of it using the test framework that machine controller uses